plugins/chunter-resources/src/components/Channel.svelte

```diff
@@ -13,19 +13,22 @@
 // limitations under the License.
 -->
 <script lang="ts">
+  import { afterUpdate, beforeUpdate, onDestroy } from 'svelte'
```

Comment:
Why you modify chunter module at all? I supposed you will modify only inbox module(notifications)